#####################################################################################################################################################
###########################################################  YOLO Model Configuration ############################################################
#####################################################################################################################################################

show_output:
  - False

yolo_model:
  model:
    - yolov4

  light_model:
    - yolov3-tiny

  # input size of the network
  input_size:
    - 320
    - 320

    #Scalar with mean values which are subtracted from channels.
  model_mean:
    - 0
    - 0
    - 0

  #Multiplier for frame values.
  model_scale:
    - 0.00392

  #Flag which indicates that swap first and last channels.
  model_swapRGB:
    - True

  #Flag which indicates whether image will be cropped after resize or not. blob(n, c, y, x) = scale * resize( frame(y, x, c) ) - mean(c) )
  model_crop:
    - False

  #A threshold used to filter boxes by confidences.
  conf_threshold:
    - 0.3

  #A threshold used in non maximum suppression.
  nms_threshold:
    - 0.4
